cmd/livepeer_cli/wizard_deposit.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func (w *wizard) deposit() {
+	fmt.Printf("Current deposit: %v\n", w.getDeposit())
+	fmt.Printf("Current token balance: %v\n", w.getTokenBalance())
+	fmt.Printf("Enter Deposit Amount - ")
+	amount := w.readInt()
+	httpGet(fmt.Sprintf("http://%v:%v/deposit?amount=%v", w.host, w.httpPort, amount))
```

Comment:
These should be POST requests right? Don't think we necessarily need to switch over now if a lot of the state changing end points already use GET requests, but we should probably come back and do so later especially if the HTTP server is used outside of the CLI wizard